dart/lib/src/sentry_template_string.dart

```diff
@@ -0,0 +1,44 @@
+class SentryTemplateString {
+  SentryTemplateString(this.template, this.arguments);
+
+  final String template;
+  final List<dynamic> arguments;
+  static final _regex = RegExp(r'%(?:%|s)');
+
+  String format() {
+    assert(arguments.isNotEmpty,
+        'No arguments provided for template with placeholders.');
```

Comment:
```suggestion
    assert(arguments.isNotEmpty,
        'No arguments provided for template.');
```

I guess since we assert for placeholder after this we can shorten this